### DISCUSSION TITLE
Welcome to Projectile's Discussions!

### DISCUSSION THREAD

**Discussion by bbatsov** [bbatsov/projectile#1629] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dchsoft1212** [comment]
Hi, I would like to use Projectile on a remote server, but the Emacs version is 24.4 and cannot be updated. Which version of Projectile should I use? 

Thanks in advance for your feedback.
Dch

**Reply by bbatsov** [reply]
Projectile 0.14 was the last version that supported Emacs 24.

**Reply by dchsoft1212** [reply]
Thank you very much, I will use it.
